plume-admin-api-log/src/main/java/com/coreoz/plume/admin/LogEntryTransformer.java

```diff
@@ -46,23 +50,56 @@ default LogEntryTransformer applyOnlyToResponsesWithHeader(String headerName, St
     }
 
     static LogEntryTransformer limitBodySizeTransformer(int bodyCharLengthLimit) {
-        return (request, response, apiTrace) -> {
+        return (request, response, apiLogEntry) -> {
             if (bodyCharLengthLimit < 0) {
-                return apiTrace;
+                return apiLogEntry;
             }
-            String bodyRequest = apiTrace.getBodyRequest();
+            String bodyRequest = apiLogEntry.getBodyRequest();
             if (bodyRequest != null && bodyRequest.length() > bodyCharLengthLimit) {
-                apiTrace.setBodyRequest(bodyRequest.substring(0, bodyCharLengthLimit));
+                apiLogEntry.setBodyRequest(bodyRequest.substring(0, bodyCharLengthLimit));
             }
-            String bodyResponse = apiTrace.getBodyResponse();
+            String bodyResponse = apiLogEntry.getBodyResponse();
             if (bodyResponse != null && bodyResponse.length() > bodyCharLengthLimit) {
-                apiTrace.setBodyResponse(bodyResponse.substring(0, bodyCharLengthLimit));
+                apiLogEntry.setBodyResponse(bodyResponse.substring(0, bodyCharLengthLimit));
             }
-            return apiTrace;
+            return apiLogEntry;
+        };
+    }
+
+    static LogEntryTransformer hideJsonFields(List<String> jsonFieldKeysToHide, String replacement) {
```

Comment:
Il faudrait ajouter une description (en anglais)

Reply:
Idéalement, il faudrait peut être donner un exemple : 
`hideJsonFields(List.of("password"), "***") will replace the JSON body `{"login": "john", "password": "azerty"}` by `{"login": "john", "password": "***"}`